<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationInput</class>
 <widget class="QDialog" name="SimulationInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextEdit" name="editor"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="LE_info">
     <property name="text">
      <string>Edit you mac input file and run the simulation:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="PB_previous">
       <property name="text">
        <string>previous</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="PB_next">
       <property name="text">
        <string>Proceed</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3"/>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Input:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_input"/>
     </item>
     <item>
      <widget class="QPushButton" name="PB_load">
       <property name="text">
        <string>Load File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="2">
    <widget class="QPushButton" name="PB_save">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
